CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## Unreleased
 ### Added
 - [#7 - Product now has condition attribute that is displayed on product detail page and is generated in google feed](https://github.com/shopsys/demoshop/pull/7)
+- [#6 - Shipping method with pickup places](https://github.com/shopsys/demoshop/pull/6)
+    - transport, Order entities and forms are extended
+    - PickUpPlace entity is added
+    - cronmodule for download of pick up places is created 
```

Comment:
I think the description "Added new shipping method Zasilkovna. Pick up places downloaded by cron." would be enough. Im not sure that "transport, Order entities and forms are extended" is not too detailed for CHANGELOG.md

Reply:
Solved